# A 2D wind vector

float64 speed
float64 direction